### ISSUE TITLE
 ERROR in main: Server Error: maximum recursion depth exceeded in comparison

### ISSUE THREAD

**Issue by EitanBe** [xlwings/jsondiff#21]
```
File "lib/python3.6/site-packages/jsondiff/__init__.py", line 597, in diff
    return cls(**kwargs).diff(a, b, fp)
  File "lib/python3.6/site-packages/jsondiff/__init__.py", line 501, in diff
    d, s = self._obj_diff(a, b)
  File "lib/python3.6/site-packages/jsondiff/__init__.py", line 488, in _obj_diff
    return self._list_diff(a, b)
  File "lib/python3.6/site-packages/jsondiff/__init__.py", line 408, in _list_diff
    for sign, value, pos, s in self._list_diff_0(C, X, Y, len(X), len(Y)):
  File "lib/python3.6/site-packages/jsondiff/__init__.py", line 374, in _list_diff_0
    for annotation in self._list_diff_0(C, X, Y, i-1, j-1):
  File "lib/python3.6/site-packages/jsondiff/__init__.py", line 374, in _list_diff_0
    for annotation in self._list_diff_0(C, X, Y, i-1, j-1):
  File "lib/python3.6/site-packages/jsondiff/__init__.py", line 374, in _list_diff_0
    for annotation in self._list_diff_0(C, X, Y, i-1, j-1):
  [Previous line repeated 958 more times]
  File "lib/python3.6/site-packages/jsondiff/__init__.py", line 372, in _list_diff_0
    d, s = self._obj_diff(X[i-1], Y[j-1])
  File "lib/python3.6/site-packages/jsondiff/__init__.py", line 484, in _obj_diff
    return self._dict_diff(a, b)
  File "lib/python3.6/site-packages/jsondiff/__init__.py", line 468, in _dict_diff
    d, s = self._obj_diff(v, w)
  File "lib/python3.6/site-packages/jsondiff/__init__.py", line 494, in _obj_diff
    return self.options.syntax.emit_value_diff(a, b, 1.0), 1.0
  File "lib/python3.6/site-packages/jsondiff/__init__.py", line 109, in emit_value_diff
    if s == 1.0:
RecursionError: maximum recursion depth exceeded in comparison
[2018-07-15 14:47:23,259] ERROR in main: Server Error: maximum recursion depth exceeded in comparison
127.0.0.1 - - [15/Jul/2018 14:47:23] "POST /compare HTTP/1.1" 500
```

**Comment by fzumstein** [comment]
see #18

**Comment by thelululand** [comment]
I still get this error

**Comment by IrisPeres** [comment]
How is this? I'm getting this error. Can you pls resolve it?

**Comment by fzumstein** [comment]
I just released 1.3.0 that should have this fix
